### ISSUE TITLE
Hard drive dead

### ISSUE THREAD

**Issue by jdigglets** [almindor/etherwall#79]
I know my password and address but while I have tried to link the old account it just wont do it. How can I link the old address on the new install to get access to my account?

**Comment by jdigglets** [comment]
Sorry I dont think I had explained properly in the first message. Once my HD blew up I just bought a new HD and reinstalled windows on it. However my geth was in the previous hd. Now when I started etherwall it made the sync to a different address. How can I make it all go back to normal. I'm sure whoever developed this wallet must have thought about something like this happening sometime to someone.

**Comment by almindor** [comment]
I understand correctly you had Etherwall/datadir on a hard-drive that died and then reinstalled on a new one. There's no "automatic backup" as Etherwall can't know what would be a safe place to store the backup to on it's own.

You can do full backup of the whole wallet by going to "wallet/export" in the main menu. You can also export individual accounts to files (encrypted with their passwords) by right clicking an account and exporting the geth account to a directory.

Etherwall also supports TREZOR hardware wallet which is probably the safest way to handle larger amounts of crypto (I use it for 95% of mine, with the rest in Etherwall and MetaMask for usage).

If you didn't do any of the backups yourself then the only option to recover the keys would be a recovery of the damaged drive. Depending on how much you had stored there it might be worth it.

Also NOTE: the default datadir/keystore location used by Geth/Etherwall is NOT where you put Etherwall itself. It might be possible the keystore was actually on a different drive. In Windows the default is on `C` drive.

**Comment by jdigglets** [comment]
Correct me if im wrong? If I dont have a backup or the keystore file I will never get access to my wallet?

**Comment by almindor** [comment]
No, sorry. You can still try to recover it from the damaged drive. There are services that do this and the success rate is higher than people think (at least for old magnetic drives, not sure about SSDs).

It's only worth it if the ETH would cover it tho.

**Comment by jdigglets** [comment]
Ok I have left the HD on a recovery specialist. Once I get it back (if I do) what is the process I need to follow with the file.

Thank for you help Almindor

**Comment by almindor** [comment]
Yes so this is in Windows correct?

The default keystore location in windows is in `C:\Users\<username>\AppData\Roaming\Ethereum\keystore`

If you didn't change the datadir setting on your original Etherewall then this is what the recovery should get. You only need the `UTC--*` files from that folder, or possibly only one, depending on which addresses had ether in them. The filenames contain the address at the end. If they can recover that then you can just copy them over to the new drive under the same folder and Etherwall/geth should pick them up again.

I'll leave the issue open for some time, if you can please report how it went, I'm curious :)
